modules/canvaskit/gpu.js

```diff
@@ -103,5 +103,15 @@
       };
       // Default to trying WebGL first.
       CanvasKit.MakeCanvasSurface = CanvasKit.MakeWebGLCanvasSurface;
+
+      // Return the texture for the surface
+      CanvasKit.Surface.prototype.getTexture = function() {
+        var textureId = this._getTextureId();
+        if (textureId === 0) {
+          return null;
+        }
+        return GL.textures[textureId];
```

Comment:
GL.textures && GL.textures[textureId] perhaps?
But then it may return undefined... 
Could you please check the correct case?

Reply:
GL.textures is provided by emscripten, so it should be available. I'll add a check though.